core/src/test/resources/reference.conf

```diff
@@ -130,3 +131,7 @@ multiCloudWorkspaces {
         leonardoWsmApplicationId = "leo"
     }
 }
+
+leonardo {
+    server = "https://leonardo.dsde-dev.broadinstitute.org"
+}
```

Comment:
this will need corresponding changes in https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/rawls/rawls.conf.ctmpl before merging this PR.